RecoPixelVertexing/Configuration/python/customizePixelTracksSoAonCPU.py

```diff
@@ -0,0 +1,61 @@
+import FWCore.ParameterSet.Config as cms
+
+def customizePixelTracksSoAonCPU(process):
+  
+  process.CUDAService = cms.Service('CUDAService',
+    enabled = cms.untracked.bool(False)
+  )
+
+  # ensure the same results when running on GPU (which supports only the 'HLT' payload) and CPU
+  process.siPixelClustersPreSplitting.cpu.payloadType = cms.string('HLT')
+
+  from RecoLocalTracker.SiPixelRecHits.siPixelRecHitSoAFromLegacy_cfi import siPixelRecHitSoAFromLegacy
+  process.siPixelRecHitsPreSplitting = siPixelRecHitSoAFromLegacy.clone(
+    convertToLegacy = True
+  )
+
+  from RecoPixelVertexing.PixelTriplets.caHitNtupletCUDA_cfi import caHitNtupletCUDA
+  process.pixelTrackSoA = caHitNtupletCUDA.clone(
+    onGPU = False,
+    pixelRecHitSrc = 'siPixelRecHitsPreSplitting'
+  )
+
+  from RecoPixelVertexing.PixelVertexFinding.pixelVertexCUDA_cfi import pixelVertexCUDA
```

Comment:
this needs to be updated in order to keep the pixelTracks and pixelVertices disantangled